### PR TITLE
Set attribute to null when saving empty strings

### PR DESCRIPTION
**Pull request by casidiablo**

### CODE REVIEW

File: src/test/java/com/scopely/mapper/SimpleFreeBuilt.java [hunk 1]
```diff
@@ -5,12 +5,15 @@
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import org.inferred.freebuilder.FreeBuilder;
 
+import javax.annotation.Nullable;
+
 @FreeBuilder
 @JsonDeserialize(builder = SimpleFreeBuilt.Builder.class)
 @DynamoDBTable(tableName = "simple_free_built")
 interface SimpleFreeBuilt {
```
Comment (divyas1102) [inline src/test/java/com/scopely/mapper/SimpleFreeBuilt.java:13]: why this name?

Comment (divyas1102) [reply src/test/java/com/scopely/mapper/SimpleFreeBuilt.java:13]: i know you didn't write this now, but just curious

Comment (casidiablo) [reply src/test/java/com/scopely/mapper/SimpleFreeBuilt.java:13]: Because is a simple object _built_ by FreeBuilder.

**Comment by casidiablo** [comment]
Merge/release when you guys want. I will get my :massage_man: so if you want this to be in right now you have to do it yourself.

**Comment by avram** [comment]
This worked locally.

**Pull request merged**